user_stories.md

```diff
@@ -0,0 +1,8 @@
+ As a user,
+  - I should be able to launch the app from welcome screen
+  - I should be able to see all flavors of ice cream from the main screen
+  -Each ice cream flavor card should include a picture, title (name of flavor), description and a button that shows where the ice cream is available
+  - When I click on the button prompting info on where ice cream is available, I will be shown a list of cards of all of  the parlors that have the flavor of ice cream I clicked on  
+  - The parlors listed will clearly display the address, phone number, rating, hours and price range
```

Comment:
We can also add numberOfFlavors. Might not need to include priceRange. Most of the shops except one, are $. One store is $$.